.ci/cico_build.sh

```diff
@@ -25,7 +25,7 @@ then
   if [ -z "${RHCHEBOT_DOCKER_HUB_PASSWORD+x}" ]; then echo "WARNING: failed to get RHCHEBOT_DOCKER_HUB_PASSWORD from jenkins-env file in centos-ci job."; else export RHCHEBOT_DOCKER_HUB_PASSWORD; fi
   set -x
   yum -y update
-  yum -y install centos-release-scl java-1.8.0-openjdk-devel git patch bzip2 golang docker subversion
```

Comment:
@sleshchenko why subversion is now required for the build ?

Reply:
because there is not subversion plugin anymore

Reply:
Or it is required by some another component?

Reply:
@sleshchenko oh, sorry overlook - I thought it was added not removed ;-)